### ISSUE TITLE
Feature Request: Allow the option to export Films / TV Shows to Kodi Library.

### ISSUE THREAD

**Issue by DitDitGitHub** [vonH/plugin.video.iplayerwww#325]
It would be great if you can add an option when you select a Film / TV Show that exports the selected media to the library in the form of .strm files.

**Comment by CaptainTK** [comment]
I don't think that is going to work. Correct me, if I am wrong, but my understanding of strm files is that they need to contain the URL of the stream itself. However, in case of iPlayer, these are not valid indefinitely. You need to get the currently valid stream using the mediaselector. That won't work from within an strm file.

What are you trying to achieve? Save programmes for later playback? Why not use the functionality the add-on already provides for that using the BBCiD?

**Comment by DitDitGitHub** [comment]
> I don't think that is going to work. Correct me, if I am wrong, but my understanding of strm files is that they need to contain the URL of the stream itself. However, in case of iPlayer, these are not valid indefinitely. You need to get the currently valid stream using the mediaselector. That won't work from within an strm file.
> 
> What are you trying to achieve? Save programmes for later playback? Why not use the functionality the add-on already provides for that using the BBCiD?

Sorry for the delayed response.
Obviously it is not necessary for this feature to be added, I think of it more as a QoL change. I like having one space for all things I would watch, and this feature would enable it.
